leetcode/src/main/java/com/wenmq/cn/leetcode/Solution594.java

```diff
@@ -0,0 +1,63 @@
+package com.wenmq.cn.leetcode;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Q594
+ * https://leetcode-cn.com/problems/longest-harmonious-subsequence/
+ */
+public class Solution594 {
+    /**
+     * 1 <= nums.length <= 2 * 10^4
+     * -10^9 <= nums[i] <= 10^9
+     */
+    public int findLHS(int[] nums) {
+        HashMap<Integer, Integer> map = new HashMap<>();
+        for (int num : nums) {
+            map.put(num, map.getOrDefault(num, 0) + 1);
+        }
+        int max = 0;
+        for (Map.Entry<Integer, Integer> entry : map.entrySet()) {
```

Comment:
nice action. 
for-each loop at entrySet insteads of keySet.
the former needs no get action.